Fix GCloud IT: test_max_messages error not received expected messages

|Related issue|
|---|
|https://github.com/wazuh/wazuh-qa/issues/3057

| Date|  Commit | Commit title | Build | |
|--|--|--|--|--|
| 2022-07-03| [2efe60e](https://github.com/wazuh/wazuh/commit/b4c9ab6680d318b1ae20d73b72d6d5450bf8f7be) | Fix remoted not updating total bytes sent by UDP (#13979)| [#28540](https://ci.wazuh.info/job/Test_integration/28540/) |  |

## Case 

| Module |  Tier| Wazuh Type | OS 
|--|--|--|--|
test_gcloud/test_functionality/test_max_messages.py | 0 | Manager| CentOS

### Description

```
23:56:04  FAILED test_gcloud/test_functionality/test_max_messages.py::test_max_messages[get_configuration0-publish_messages0]
23:56:04  FAILED test_gcloud/test_functionality/test_max_messages.py::test_max_messages[get_configuration0-publish_messages1]
23:56:04  FAILED test_gcloud/test_functionality/test_max_messages.py::test_max_messages[get_configuration0-publish_messages2]
```

## Tasks
- [x] Run tests in Jenkins
- [x] Fix failures

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -29,6 +29,7 @@ Release report: TBD
 
 ### Changed
 
+- Fix GCloud IT: test_max_messages error not received expected messages - ([#3083](https://github.com/wazuh/wazuh-qa/pull/3083)) \- (Tests) 
 - Update syscollector deltas integration tests ([#2921](https://github.com/wazuh/wazuh-qa/pull/2921)) \- (Tests)
 - Update deprecated WDB commands ([#2966](https://github.com/wazuh/wazuh-qa/pull/2966)) \- (Tests)
 - Move the 'get_datetime_diff' function to 'wazuh-testing' utils module ([#2782](https://github.com/wazuh/wazuh-qa/pull/2782)) \- (Framework + Tests)

--- a/deps/wazuh_testing/wazuh_testing/gcloud.py
+++ b/deps/wazuh_testing/wazuh_testing/gcloud.py
@@ -57,7 +57,7 @@ def detect_gcp_start(file_monitor):
 
 
 def callback_received_messages_number(line):
-    received_messages_number = rf".*wm_gcp_pubsub_run\(\): INFO: - INFO - Received and acknowledged (\d+) messages"
+    received_messages_number = r'.*wm_gcp_parse_output\(\): INFO: Received and acknowledged (\d+) messages'
     match = re.findall(received_messages_number, line)
     if match:
         return match

